yastd/dbus_api.md

```diff
@@ -0,0 +1,29 @@
+
+## get the list of available languages
+
+Iface: o.o.YaST.Installer1.Language
+
+stupid api:
+  AvailableLanguages -> array(string)
```

Comment:
what about using ObjectManager interface for it?

Reply:
ObjectManager is an implementation detail about the tree of objects and their interfaces and properties.
Your suggestion implies the existence of the objects
```
/org/opensuse/YaST/Installer1/Language/Available/af_ZA
/org/opensuse/YaST/Installer1/Language/Available/ar_EG
/org/opensuse/YaST/Installer1/Language/Available/bg_BG
...
```

It's one of the options, I still don't know if it's the best one

Reply:
yeah, for sure iterating over all that objects can be performance kill, so maybe returnning array of structs with open ended map as the last element or array of map can be better solution that is extendable and also good performance wise.....But as mentioned we should first clarify what is an exact goal of this API.